packages/icons/src/svg/recreation.svg

```diff
@@ -1,4 +1,3 @@
-<svg width="25" height="24" viewBox="0 0 25 24" fill="none" xmlns="http://www.w3.org/2000/svg">
-<path d="M23.6767 7.62828L12.4256 0.127513C12.1756 -0.0425043 11.8455 -0.0425043 11.5955 0.127513L0.334371 7.62828C-0.00566388 7.8583 -0.105674 8.32835 0.124349 8.66838C0.354373 9.00842 0.824421 9.10843 1.16446 8.8784L12.0056 1.64767L22.8367 8.8684C22.9667 8.95841 23.1167 8.99842 23.2567 8.99842C23.4967 8.99842 23.7368 8.8784 23.8768 8.66838C24.1068 8.31835 24.0168 7.8583 23.6767 7.62828Z" fill="#171717"/>
-<path d="M18.7562 13.4989H14.0358C13.0557 13.4989 12.1056 13.7689 11.2755 14.289C10.4454 14.799 9.77533 15.5391 9.33529 16.4092L8.98525 17.1092C8.94525 16.1791 8.56521 15.2691 7.90514 14.609C7.19507 13.8989 6.25497 13.5089 5.25487 13.5089H3.00464V12.7588C3.00464 12.3388 2.66461 11.9987 2.25457 11.9987C1.84452 11.9987 1.50449 12.3388 1.50449 12.7488V23.2499C1.50449 23.6599 1.84452 23.9999 2.25457 23.9999H21.7566C22.1666 23.9999 22.5066 23.6599 22.5066 23.2499V17.2493C22.5066 16.2592 22.1066 15.2991 21.4065 14.599C20.6964 13.8889 19.7563 13.4989 18.7562 13.4989ZM10.6854 17.0692C10.9955 16.4492 11.4755 15.9191 12.0656 15.5591C12.6556 15.189 13.3457 14.999 14.0358 14.999H18.7562C19.3563 14.999 19.9264 15.229 20.3464 15.6591C20.7665 16.0791 21.0065 16.6592 21.0065 17.2493V17.9993H10.2154L10.6854 17.0692ZM5.25487 14.999C5.85493 14.999 6.42499 15.229 6.84503 15.6591C7.26508 16.0791 7.5051 16.6592 7.5051 17.2493V17.9993H3.00464V14.999H5.25487ZM21.0065 22.4998H3.00464V19.4995H21.0065V22.4998Z" fill="#171717"/>
+<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
This icon has been changed – discuss with Jirka B. if it's a desired change or not.

Reply:
It's not a bug, it's a feature!